.github/ISSUE_TEMPLATE/user_registration.yml

```diff
@@ -0,0 +1,54 @@
+name: "Support Pants - List My Company on pantsbuild.org"
+description: Add a company or organization to the public list of Pants Users
+title: "Please add XXXXXX to the public list of Pants Users"
+labels: [user-registration]
+assignees:
+  - benjyw
+  - cczona
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping grow the Pants community by being featured on the public list of Pants Users!
+  - type: input
+    id: name
+    attributes:
+      label: "Company name"
+      description: The name of your company/organization.
+      placeholder: e.g., Acme
+    validations:
+      required: true
+  - type: input
+    id: website
+    attributes:
+      label: "Company website"
+      description: Your company's main website
+      placeholder: e.g., https://www.acme.com
+    validations:
+      required: true
+  - type: input
+    id: email
+    attributes:
+      label: Contact email
+      description: |
+        How can we contact you if we have questions about this form? We *won't* publish your
+        email address on the Pants Users list, or use it for any other purpose.
+      placeholder: e.g., wilecoyote@acme.com
+    validations:
+      required: true
```

Comment:
I don't think that we strictly need to ask for an email address here, given that we can follow up on the created github issue to ask further questions. I expect that a non-trivial number of users will be uncomfortable with (or perhaps surprised by) having their email address show up in the issue.

Reply:
Fixed.